validate_sample.py

```diff
@@ -18,41 +19,47 @@ class SampleValidator:
 
     ESTAB_TYPES = {'HALL OF RESIDENCE', 'CARE HOME', 'HOSPITAL', 'HOSPICE', 'MENTAL HEALTH HOSPITAL',
                    'MEDICAL CARE OTHER', 'BOARDING SCHOOL', 'LOW/MEDIUM SECURE MENTAL HEALTH',
-                   'HIGH SECURE MENTAL HEALTH', 'HOTEL', 'YOUTH HOSTEL', 'HOSTEL', 'MILITARY SLA', 'MILITARY US',
+                   'HIGH SECURE MENTAL HEALTH', 'HOTEL', 'YOUTH HOSTEL', 'HOSTEL', 'MILITARY SLA', 'MILITARY US SLA',
                    'RELIGIOUS COMMUNITY', 'RESIDENTIAL CHILDRENS HOME', 'EDUCATION OTHER', 'PRISON',
                    'IMMIGRATION REMOVAL CENTRE', 'APPROVED PREMISES', 'ROUGH SLEEPER', 'STAFF ACCOMMODATION',
                    'CAMPHILL', 'HOLIDAY PARK', 'HOUSEHOLD', 'SHELTERED ACCOMMODATION', 'RESIDENTIAL CARAVAN',
                    'RESIDENTIAL BOAT', 'GATED APARTMENTS', 'MOD HOUSEHOLDS', 'FOREIGN OFFICES', 'CASTLES', 'GRT SITE',
-                   'MILITARY SFA', 'EMBASSY', 'ROYAL HOUSEHOLD', 'CARAVAN SITE', 'MARINA', 'TRAVELLING PERSONS',
-                   'TRANSIENT PERSONS'}
+                   'MILITARY SFA', 'EMBASSY', 'ROYAL HOUSEHOLD', 'CARAVAN', 'MARINA', 'TRAVELLING PERSONS',
+                   'TRANSIENT PERSONS', 'MIGRANT WORKERS', 'MILITARY US SFA'}
 
     def __init__(self):
         self.schema = {
-            'UPRN': [mandatory(), max_length(12), numeric(), no_padding_whitespace()],
-            'ESTAB_UPRN': [mandatory(), max_length(12), numeric(), mandatory(), no_padding_whitespace()],
+            'UPRN': [mandatory(), max_length(13), numeric(), no_padding_whitespace()],
+            'ESTAB_UPRN': [mandatory(), max_length(13), numeric(), no_padding_whitespace()],
             'ADDRESS_TYPE': [mandatory(), in_set({'HH', 'CE', 'SPG'}), no_padding_whitespace()],
-            'ESTAB_TYPE': [mandatory(), in_set(self.ESTAB_TYPES), max_length(255), no_padding_whitespace()],
+            'ESTAB_TYPE': [mandatory(), in_set(self.ESTAB_TYPES), no_padding_whitespace()],
```

Comment:
doesn't need to check for padding/whitespace because it already has an `in_set` check